client/src/components/Header.js

```diff
@@ -28,6 +27,15 @@ gap:3rem;
 
 
 
+
+
+
+
+
```

Comment:
this empty space makes your file bigger and harder to read. there are good tools, which automatically format your code in a nice and clean way, for example https://prettier.io/
the implementation is not to hard, you get a good benefit from it =)